### PR TITLE
Third-party bumps

### PR DESCRIPTION
**Pull request by NiLuJe**
Fingers crossed we can finally update harfbuzz ;o).

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1631)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]

**Comment by Frenzie** [comment]
Heh, I figured we'd just wait for 8.0.1 in a week or two.

**Comment by Frenzie** [comment]
It sounds like there's an issue with the libjpegturbo thing?

https://app.circleci.com/pipelines/github/koreader/koreader-base/1670/workflows/bb66a69d-6bcc-41ca-9cfa-2ab30bdc6baf/jobs/14997?invite=true#step-105-1073

File: ffi/jpeg.lua [hunk 1]
```diff
@@ -95,9 +95,8 @@ function Jpeg.encodeToFile(filename, source_ptr, w, stride, h, quality, color_ty
     end
 
     turbojpeg.tj3Destroy(handle)
-    if jpeg_image[0] ~= nil then
-        turbojpeg.tj3Free(jpeg_image[0])
-    end
+    -- Handles NULL pointers sanely
+    turbojpeg.tj3Free(jpeg_image[0])
```
Comment (Frenzie) [inline ffi/jpeg.lua]: This change isn't dangerous?

Comment (NiLuJe) [reply ffi/jpeg.lua]: Nope, the comment was supposed to help avoid similar questions in the future ^^.

```
 * @param buffer address of the buffer to free.  If the address is NULL, then
 * this function has no effect.
```

c.f., https://rawcdn.githack.com/libjpeg-turbo/libjpeg-turbo/main/doc/html/group___turbo_j_p_e_g.html#gaddb84fb6c81769e9faa0f5a63b296606

**Pull request merged**